src/borda_clj/client.clj

```diff
@@ -4,26 +4,35 @@
             [byte-streams :as bs]
             [cheshire.core :as json]))
 
-(def discard_key {:op "_discard"})
+(def measure_key {:op "_measure"})
 
-(defn empty-measurements []
-  {discard_key {:success_count 0}})
+(defn measure_succeeded [count_measure measurements]
+  (if (not count_measure)
+    measurements
+    (update measurements measure_key (partial merge-with +) {:success_count 1})))
```

Comment:
In this case we can't just use `update-in` as suggested in my last comment, because we'd get a NullPointerException if the key is not there.